locale-source/locale/de/LC_MESSAGES/djangojs.po

```diff
@@ -234,13 +232,13 @@ msgstr ""
 #: adhocracy4/comments_async/static/comments_async/ai_report.jsx:13
 #: adhocracy4/comments_async/static/comments_async/ai_report.jsx:13
 msgctxt "defakts"
-msgid "Show info to users"
+msgid "Show AI info to users"
```

Comment:
@TeaJayyy the defakts translations come from A4. Need to remove them manually in the future before pushing to transifex.